deploy/crds/pulumi.com_stacks.yaml

```diff
@@ -57,6 +57,11 @@ spec:
               items:
                 type: string
               type: array
+            envSecretsFromPath:
```

Comment:
These are strictly secrets, are they?  In the sense that the other uses of these he word `Secret` here are about K8s `Secret` objects. 

Are there other precedents for naming for things like this in other operators we could model after?  Not sure if this pattern has common terminology associated with it?

Reply:
Fair point. I was positioning this as an alternative to `envSecrets`. It can just be called `envFromPath` instead.

I was also wondering if it makes sense to take a slightly different tack on this by modeling things as a reference instead. For instance for #97 I can foresee us wanting to support literal, secret reference, environment variable or a file as well and could reuse the same model. As a result I am proposing adding a `ResourceRef` type. Other operators seem to take a similar approach.

See 311e6aad3755faf4220fb392ab9dda2aa89ec471 for prototyped specification. If you agree with this approach, I can replace the current implementation to use the above.